stage/one/s8_dict.py

```diff
@@ -0,0 +1,55 @@
+from utils.form import blank_form
+from utils import fields_generate
+
+route = {
+    'type': blank_form, 
+    'url': '/stage/one/dict',
+    'methods': [ 'GET', 'POST' ]
+}
+data = {
+    'title': 'Python dictionaries',  
+    'image': 'https://community-cdn-digitalocean-com.global.ssl.fastly.net/assets/tutorials/images/large/EBOOK_PYTHON_no-name.png?1516826609',
+    'description': [
+        'Lets get familiar with Python diciontaries (dict).',
+        'In Python, dictionary is a collection of key value pairs, ' +
+        'such that each key of a given type maps to a value of a given type.',
+        'You can reference dictionary methods here: https://docs.python.org/3/' +
+        'tutorial/datastructures.html#dictionaries',
+        '\n',
+        'Now lets work on some exercises. You have a following ' +
+        'array of integers: [1, 1, 2, 1, 1, 2, 2]',
+        'Use a dictionary to count the frequency of each element ' +
+        'do not use any other data structures.'
+    ],
+    'code': [
+        'l = [1, 1, 2, 1, 1, 2, 2]',
+        'counter = {}',
+        'for i in l:',
+        '   if i in counter:',
+        '       _____[i] += 1',
+        '   else:',
+        '       _____[i] = 1',
+        'print(str(counter))'
+    ],
+    'fields': []
+}
+
+data['fields'] = fields_generate(data)  # NEVER remove this line!!
+
+async def sanic_request(request):
+    try:
+        return override(request)
+    except NameError:
+        global data, route
+        return route['type'](data, request, answer)
+
+def answer(stdout, stderr):
+    try:
+        if stderr != []:
+            return stderr
```

Comment:
Here should send a False.
You can check this [part](https://github.com/YunOSC/PyFun-Backend/blob/master/utils/form/__init__.py#L15,#L23).